content/tutorials/portenta-h7/vs-openmv-ttn/content.md

```diff
@@ -0,0 +1,184 @@
+# Connecting the Vision Shield to TTN Using LoRa
```

Comment:
How will we differentiate the Arduino version from the OpenMV version in the list of tutorials?

Reply:
What title do you think is better, "Connecting the Vision Shield to TTN Using LoRa and OpenMV" or "Use OpenMV to Connect the Vision Shield to TTN Using LoRa"?

Reply:
The shorter the better. @Lenardgeorge any suggestions?

Reply:
Then we say **Connecting the Vision Shield to The Things Network using OpenMV**  ;)